src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/ManifestVersionUpdate.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace Microsoft.NET.Sdk.WorkloadManifestReader
 {
+    //  TODO: Do we need this class, or the existing version information anymore now that workload manifest are side by side?
```

Comment:
The only time I can think of that it would be helpful is if we're rolling back to a pinned version of something, but I don't think we handle that 100% correctly now, so we can probably dispense with it.